### ISSUE TITLE
SHIP #3437 #3438 - D 32 Daring/Dragon [Type 45 Batch 1] (United Kingdom - 2023) #4139 #3200 - D 32 Daring/Dragon [Type 45 Batch 1] (United Kingdom - 2023) SSM rationalisation   

### ISSUE THREAD

**Issue by KLABDB3K** [PygmalionOfCyprus/cmo-db-requests#4534]
### DB Selector

DB3K

### Affected DBID(s)

Ship #4139 #3437 #3438 #3200 

### Summary of Changes

These RN Type 45 entries appear to be hybrids blending aspects of the real warship prospective configuration:
#3437 - D 32 Daring [Type 45 Batch 1] (United Kingdom - 2023) has CAMM but not NSM, - Still using Harpoon.
#4139 - D 32 Daring [Type 45 Batch 1] (United Kingdom - 2023) has NSM but not CAMM 
#3438 - D 35 Dragon [Type 45 Batch 2] (United Kingdom - 2023) has CAMM but not NSM, - Still using Harpoon.
#3200 - D 35 Dragon [Type 45 Batch 2] (United Kingdom - 2023) has NSM but not CAMM 

Would benefit from a rationalised combination vessel which has both CAMM and NSM -(No Harpoon) circa 2026 perhaps please? 

### Sources

DB3K 505

**Comment by FrangibleCover** [comment]
I believe, although this is a vague impression more than knowledge, the current ships have CAMM and no SSM at all and in the future the NSM will be fitted. Therefore 4139 and 3200 should be given CAMM and shunted out to the future and the others should have Harpoon removed.

https://www.navylookout.com/what-is-the-state-of-royal-navy-land-attack-missile-capability/ - This supports what I think but is also just a random guy on the internet.
https://www.royalnavy.mod.uk/news-and-latest-activity/news/2022/november/22/20221122-royal-navy-warships-to-receive-harpoon-replacement-from-next-year - A read between the lines of this says Harpoon is gapped and NSM is going to be on everything before 2028.

**Comment by KLABDB3K** [comment]
Yes agreed ref no SSM but with CAMM, think Harpoon is effectively "out of warranty" for want of a more lengthy description?

**Comment by KMPDB** [comment]
Added DB519.
